# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/referencing/hotfix-license.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/pyproject.toml.orig	2000-01-01 02:00:00.000000000 +0200
+++ b/pyproject.toml	2025-02-02 17:58:40.108373672 +0200
@@ -10,8 +10,7 @@
 description = "JSON Referencing + Python"
 requires-python = ">=3.9"
 readme = "README.rst"
-license = "MIT"
-license-files = ["COPYING"]
+license = { text = "MIT" }
 keywords = ["json", "referencing", "jsonschema", "openapi", "asyncapi"]
 authors = [
   { name = "Julian Berman", email = "Julian+referencing@GrayVines.com" },
@@ -38,6 +37,9 @@
   "typing-extensions>=4.4.0; python_version<'3.13'",
 ]
 
+[project.license-files]
+paths = ["COPYING"]
+
 [project.urls]
 Documentation = "https://referencing.readthedocs.io/"
 Homepage = "https://github.com/python-jsonschema/referencing"
